### ISSUE TITLE
Add ability to Skip Songs

### ISSUE THREAD

**Issue by fankijo** [theothernt/AerialViews#187]
Hey Neil, awesome App!!!! It would be cool of you could add Play/pause as well as next and previous Song as actions for the D-Pad Buttons.

Thanks and keep going woth tbis Great App!

**Comment by theothernt** [comment]
I can look into how this would work, but you'd either have to change a) "skip next video" to "skip next track" or b) use the long press-press actions - is that what you were thinking?

**Comment by fankijo** [comment]
Yeah thats exactly what i thought about. Just to be able to choose the music Controls in the D-Pad Actions.

**Comment by theothernt** [comment]
Ok, I was looking into this a bit more. I did come up with a couple of options...

1) Add 'Skip to next music track' and  'Skip to previous music track' options for Press and Press + Hold
2) Add an option (enable/disable) to 'When you skip a video, it will also skip the current music track as well"

Does the 2nd option make sense or do you think it should be a separate action?

**Comment by fankijo** [comment]
Yeah the first option is exactly what I thought of.
2nd option doesnt make sense to me.

**Comment by theothernt** [comment]
Quick question - do you also use the Now Playing feature to display the name of the current song?

**Comment by fankijo** [comment]
Yes

**Comment by theothernt** [comment]
I've just released v1.7.5 beta3. You can test it by signing up at...

[https://play.google.com/apps/testing/com.neilturner.aerialviews](https://play.google.com/apps/testing/com.neilturner.aerialviews)

**Comment by fankijo** [comment]
Great, will test this later!

**Comment by fankijo** [comment]
First wondered why I cant see any differences, but after switching language to english I could see the new options for the D-Pad Buttons. Does exactly what I wanted it to do and works fine.

Great!

**Comment by theothernt** [comment]
That's great, thank you for testing the feature.

Sorry about the missing language text, that will be added in the next beta - English for the moment, everything should be translated when v1.7.5 is released in a few weeks.

**Comment by theothernt** [comment]
I've release another beta, that should fix the issue with the text - any untranslated text will be in English for the moment.
